activerecord/test/cases/autosave_association_test.rb

```diff
@@ -597,7 +597,8 @@ def self.name; "Person"; end
     assert_predicate reference_valid, :valid?
     assert_not_predicate reference_invalid, :valid?
     assert_not_predicate p, :valid?
-    assert_equal [{ error: "should be favorite" }], p.errors.details[:"references[1]"]
+    assert_equal [{ error: "should be favorite" }], p.errors.details[:"references[1].base"]
+    assert_equal "should be favorite", p.errors[:"references[1].base"].first
```

Comment:
Are we testing all the things that were reported as broken now? Just so we can avoid future regressions. 😄

Reply:
Now, yes